hackathon_site/dashboard/frontend/src/testing/mockData.tsx

```diff
@@ -923,3 +923,5 @@ export const teamsList = [
         Members: ["Name1", "Name2", "Name3", "Name4"],
     },
 ];
+
+export const mockTeams: Team[] = [mockTeam, mockValidTeam];
```

Comment:
probs should do `[...mockTeam, ...mockValidTeam]` so it makes a copy of the team objects.

Reply:
done

Reply:
hm this breaks it